packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -512,17 +513,30 @@ export class LitNodeClientNodeJs
     resourceAbilityRequests: LitResourceAbilityRequest[];
   }): Promise<boolean> => {
     const authSigSiweMessage = new SiweMessage(authSig.signedMessage);
-
-    try {
-      await authSigSiweMessage.validate(authSig.sig);
-    } catch (e) {
-      console.debug('Need retry because verify failed', e);
-      return true;
+    // We will either have `ed25519` or `LIT_BLS` as we have deviated from the specification of SIWE and use BLS signatures in some cases
+    // Here we need to check the `algo` of the SIWE to confirm we can validate the signature as if we attempt to validate the BLS signature here
+    // it will fail.
+    if (authSig.algo === `ed25519`) {
+      try {
+        await authSigSiweMessage.validate(authSig.sig);
+      } catch (e) {
+        return true;
+      }
+    } else if (authSig.algo === `LIT_BLS`) {
+      try {
+        blsSessionSigVerify(
+          blsSdk.verify_signature,
```

Comment:
I didnt want to bind directly to the helper right now so instead went with the lower level primitive since it is used for the bls session signature combining.

Reply:
Once the wasm refactor is merged see #503 this can be refactored to use `crypto` over `blsSdk`